client/packages/dashboard/src/widgets/DistributionWidget.tsx

```diff
@@ -116,6 +116,16 @@ export const DistributionWidget: React.FC = () => {
                     .addQuery({ status: RequisitionNodeStatus.New })
                     .build(),
                 },
+                {
+                  label: t('label.emergency'),
```

Comment:
Have left this as emergency for succinctness, but maybe needs to be more specific to show that it is emergcy AND automatically created AND new?